### PR TITLE
get current event

### PR DESCRIPTION
**Pull request by bketelsen**

### CODE REVIEW

File: pages/index.js [hunk 1]
```diff
@@ -66,6 +66,7 @@ class Index extends React.Component {
   render() {
 
     const { source, frontMatter, edata } = this.props;
+    console.log(edata)
```
Comment (perrito666) [inline pages/index.js]: leftover?

File: pages/index.js [hunk 2]
```diff
@@ -588,9 +589,9 @@ export const getStaticProps = async ({ params }) => {
   const contentFilePath = path.join(CONTENT_PATH, `index.mdx`)
   const source = fs.readFileSync(contentFilePath)
   const { content, data } = matter(source)
-  var client = new Client("azure");
+  var client = new Client("dev");
```
Comment (perrito666) [inline pages/index.js]: what does this change implies?

Comment (bketelsen) [reply pages/index.js]: this is why the build is failing actually, it is trying to hit the local encore instance.  this parameter determines which encore instance to hit.

File: pages/index.js (hunk 1 above)
Comment (bketelsen) [reply pages/index.js]: yep

**Comment by bketelsen** [comment]
This depends on showrunner #6

**Pull request merged**